core/src/main/resources/i18n/displayStrings_de.properties

```diff
@@ -1961,7 +1961,7 @@ showWalletDataWindow.includePrivKeys=Private Schlüssel einbeziehen
 tacWindow.headline=Nutzervereinbarung
 tacWindow.agree=Ich stimme zu
 tacWindow.disagree=Ich stimme nicht zu und beende
-tacWindow.arbitrationSystem=Vermittlersystem
+tacWindow.arbitrationSystem=Dispute resolution
```

Comment:
Why is the German translation (and others, see below) reverted to English?

Reply:
I looked into this briefly in Transifex and the string did change from "Vermittlersystem" to "Streitbeilegung" about a month ago but it seems that something went wrong when importing the new strings into the git repo.